index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Sports Summit India 2023 official website for registration and updates">
+    <link rel="apple-touch-icon" sizes="180x180" href="assets/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="assets/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="assets/favicon-16x16.png">
+    <link rel="manifest" href="assets/site.webmanifest">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Sports-Summit 2023</title>
+</head>
+
+<body>
+    <section id="home">
+        <header>
+            <nav class="up-nav">
+                <a href="#">
+                    <i class="fa fa-facebook" aria-hidden="true"></i>
+                </a>
+                <a href="#">
+                    <i class="fa fa-twitter" aria-hidden="true"></i>
+                </a>
+                <a href="#">
+                    English
+                </a>
+                <a href="#">
+                    My Page
+                </a>
+                <a href="#">
+                    Logout
+                </a>
+            </nav>
+            
+
+            <div class="burger">
+                <i class="fa fa-bars fa-3x" aria-hidden="true"></i>
+            </div>
+            <nav class="menu-mobile">
+                <ul class="nav-links-mobile">
+                    <li>
+                        <a href="about.html" class="a-links">About</a>
+                    </li>
+                    <li>
+                        <a href="#venue" class="a-links">Schedule</a>
+                    </li>
+                    <li>
+                        <a href="#HOF" class="a-links">Speakers</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Partner</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Join Us!</a>
+                    </li>
+                    <li>
+                        <div class="close-icon">
+                            <i class="fa fa-times fa-3x" aria-hidden="true"></i>
+                        </div>
+                    </li>
+                </ul>
+            </nav>
```

Comment:
- [ ] As pointed earlier, this button is still missing from your design. Kindly implement it. 👍 

## Behance design

<img width="451" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235325090-3f6207fb-3353-46da-aac3-7d69ceec698f.png">